docker/app/alpine/3.7/Dockerfile

```diff
@@ -18,11 +18,15 @@ RUN apk add --no-cache --update --virtual build \
         postgresql-dev
 
 WORKDIR /app
-COPY pyproject.toml /app/pyproject.toml
-RUN python3 -m pip install poetry \
-    && python3 -m poetry install --extras deploy \
+
+COPY Pipfile /app/Pipfile
+COPY Pipfile.lock /app/Pipfile.lock
+RUN python3 -m pip install pipenv \
+    && python3 -m pipenv install --dev --system --deploy \
     && apk del --purge build
 
 COPY . .
 
+RUN python3 manage.py collectstatic --no-input --clear
```

Comment:
I don't see why we would collect static files in the container. uWSGI isn't built with appropriate mimetypes, and thus can't serve these properly at the moment. I believe the plan was to put the site behind NGINX which would then do the job of serving static files.

Reply:
I'd prefer to just use [Whitenoise](http://whitenoise.evans.io/en/stable/) to serve static files.